SMLHelper/Interfaces/IEquipmentHandler.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace SMLHelper.V2.Interfaces
+{
+    /// <summary>
+    /// A handler related to EquipmentTypes
+    /// </summary>
+    public interface IEquipmentHandler
+    {
+        /// <summary>
+        /// Registers an equipment type for use when creating a equipment
+        /// </summary>
+        /// <param name="equipmentName">The name of the new equipment type</param>
+        /// <param name="displayName">The display name of the new equipment type</param>
```

Comment:
The `displayName` parameter xml doc is no longer necessary since it's gone.
Also i just noticed the `ModdedEquipmentExists` method is missing from the `IEquipmentHandler` interface. add it there, and also rename the `ModdedEquipmentExists` to `ModdedEquipmentTypeExists` for consistency and clarity reasons. then hopefully we're good to merge.

Reply:
I've removed the `displayName` xml doc line, added that method to the interface, added appropriate xml documentation for that method, and changed the method name to ensure consistency.